### DISCUSSION TITLE
No Method Error

### DISCUSSION THREAD

**Discussion by can-tls** [solidusio/solidus#4628] [Troubleshooting]
Sorry folks but I'm getting frustrated here.. 
my solidus shop was working fine, I committed and merged the branch,
made a new branch to fix something, fetched, rebased as usuall and now the shop index gives me this error:
Showing /root/code/Projekt/app/views/spree/products/_product-card.html.erb where line #21 raised:

undefined method `processed' for nil:NilClass

        }.merge(transformation)).processed

Extracted source (around line #15)

#13  def call
#14    if image
#15      image_tag image.url(size), default_options.merge(options)
#16    else
#17      content_tag :div, nil, class: ['image-placeholder', size].join(' ')
#18    end

I don't get it, I suck at coding :(

**Comment by can-tls** [comment]
the Full trace:

[solidus_core (3.2.2) app/models/concerns/spree/active_storage_adapter/attachment.rb:38:in `variant'](http://localhost:3000/shop#)
[solidus_core (3.2.2) app/models/concerns/spree/active_storage_adapter/attachment.rb:28:in `url'](http://localhost:3000/shop#)
[solidus_core (3.2.2) app/models/concerns/spree/active_storage_adapter.rb:109:in `url'](http://localhost:3000/shop#)
[app/components/image_component.rb:15:in `call'](http://localhost:3000/shop#)
[view_component (2.71.0) lib/view_component/compiler.rb:134:in `block in render_template_for'](http://localhost:3000/shop#)
[concurrent-ruby (1.1.10) lib/concurrent-ruby/concurrent/atomic/read_write_lock.rb:78:in `with_read_lock'](http://localhost:3000/shop#)
[view_component (2.71.0) lib/view_component/compiler.rb:104:in `with_read_lock'](http://localhost:3000/shop#)
[view_component (2.71.0) lib/view_component/compiler.rb:132:in `render_template_for'](http://localhost:3000/shop#)
[view_component (2.71.0) lib/view_component/base.rb:140:in `render_in'](http://localhost:3000/shop#)
[actionview (7.0.3.1) lib/action_view/helpers/rendering_helper.rb:42:in `render'](http://localhost:3000/shop#)
[app/views/spree/products/_product-card.html.erb:21](http://localhost:3000/shop#)
[actionview (7.0.3.1) lib/action_view/helpers/cache_helper.rb:175:in `cache'](http://localhost:3000/shop#)
[app/views/spree/products/_product-card.html.erb:17](http://localhost:3000/shop#)
[actionview (7.0.3.1) lib/action_view/base.rb:244:in `public_send'](http://localhost:3000/shop#)
[actionview (7.0.3.1) lib/action_view/base.rb:244:in `_run'](http://localhost:3000/shop#)
[actionview (7.0.3.1) lib/action_view/template.rb:157:in `block in render'](http://localhost:3000/shop#)
[activesupport (7.0.3.1) lib/active_support/notifications.rb:206:in `block in instrument'](http://localhost:3000/shop#)
[activesupport (7.0.3.1) lib/active_support/notifications/instrumenter.rb:24:in `instrument'](http://localhost:3000/shop#)
[activesupport (7.0.3.1) lib/active_support/notifications.rb:206:in `instrument'](http://localhost:3000/shop#)
[actionview (7.0.3.1) lib/action_view/template.rb:361:in `instrument_render_template'](http://localhost:3000/shop#)
[actionview (7.0.3.1) lib/action_view/template.rb:155:in `render'](http://localhost:3000/shop#)
[actionview (7.0.3.1) lib/action_view/renderer/partial_renderer.rb:251:in `block in render_partial_template'](http://localhost:3000/shop#)
[activesupport (7.0.3.1) lib/active_support/notifications.rb:206:in `block in instrument'](http://localhost:3000/shop#)
[activesupport (7.0.3.1) lib/active_support/notifications/instrumenter.rb:24:in `instrument'](http://localhost:3000/shop#)
[activesupport (7.0.3.1) lib/active_support/notifications.rb:206:in `instrument'](http://localhost:3000/shop#)
[actionview (7.0.3.1) lib/action_view/renderer/partial_renderer.rb:246:in `render_partial_template'](http://localhost:3000/shop#)
[actionview (7.0.3.1) lib/action_view/renderer/partial_renderer.rb:237:in `render'](http://localhost:3000/shop#)
[actionview (7.0.3.1) lib/action_view/renderer/renderer.rb:81:in `render_partial_to_object'](http://localhost:3000/shop#)
[actionview (7.0.3.1) lib/action_view/renderer/renderer.rb:53:in `render_partial'](http://localhost:3000/shop#)
[actionview (7.0.3.1) lib/action_view/helpers/rendering_helper.rb:44:in `render'](http://localhost:3000/shop#)
[app/views/spree/products/_products-grid.html.erb:13](http://localhost:3000/shop#)
[activerecord (7.0.3.1) lib/active_record/relation/delegation.rb:88:in `each'](http://localhost:3000/shop#)
[activerecord (7.0.3.1) lib/active_record/relation/delegation.rb:88:in `each'](http://localhost:3000/shop#)
[app/views/spree/products/_products-grid.html.erb:12](http://localhost:3000/shop#)
[actionview (7.0.3.1) lib/action_view/base.rb:244:in `public_send'](http://localhost:3000/shop#)
[actionview (7.0.3.1) lib/action_view/base.rb:244:in `_run'](http://localhost:3000/shop#)
[actionview (7.0.3.1) lib/action_view/template.rb:157:in `block in render'](http://localhost:3000/shop#)
[activesupport (7.0.3.1) lib/active_support/notifications.rb:206:in `block in instrument'](http://localhost:3000/shop#)
[activesupport (7.0.3.1) lib/active_support/notifications/instrumenter.rb:24:in `instrument'](http://localhost:3000/shop#)
[activesupport (7.0.3.1) lib/active_support/notifications.rb:206:in `instrument'](http://localhost:3000/shop#)
[actionview (7.0.3.1) lib/action_view/template.rb:361:in `instrument_render_template'](http://localhost:3000/shop#)
[actionview (7.0.3.1) lib/action_view/template.rb:155:in `render'](http://localhost:3000/shop#)
[actionview (7.0.3.1) lib/action_view/renderer/partial_renderer.rb:251:in `block in render_partial_template'](http://localhost:3000/shop#)
[activesupport (7.0.3.1) lib/active_support/notifications.rb:206:in `block in instrument'](http://localhost:3000/shop#)
[activesupport (7.0.3.1) lib/active_support/notifications/instrumenter.rb:24:in `instrument'](http://localhost:3000/shop#)
[activesupport (7.0.3.1) lib/active_support/notifications.rb:206:in `instrument'](http://localhost:3000/shop#)
[actionview (7.0.3.1) lib/action_view/renderer/partial_renderer.rb:246:in `render_partial_template'](http://localhost:3000/shop#)
[actionview (7.0.3.1) lib/action_view/renderer/partial_renderer.rb:237:in `render'](http://localhost:3000/shop#)
[actionview (7.0.3.1) lib/action_view/renderer/renderer.rb:81:in `render_partial_to_object'](http://localhost:3000/shop#)
[actionview (7.0.3.1) lib/action_view/renderer/renderer.rb:53:in `render_partial'](http://localhost:3000/shop#)
[actionview (7.0.3.1) lib/action_view/helpers/rendering_helper.rb:44:in `render'](http://localhost:3000/shop#)
[app/views/spree/products/_products.html.erb:24](http://localhost:3000/shop#)
[actionview (7.0.3.1) lib/action_view/base.rb:244:in `public_send'](http://localhost:3000/shop#)
[actionview (7.0.3.1) lib/action_view/base.rb:244:in `_run'](http://localhost:3000/shop#)
[actionview (7.0.3.1) lib/action_view/template.rb:157:in `block in render'](http://localhost:3000/shop#)
[activesupport (7.0.3.1) lib/active_support/notifications.rb:206:in `block in instrument'](http://localhost:3000/shop#)
[activesupport (7.0.3.1) lib/active_support/notifications/instrumenter.rb:24:in `instrument'](http://localhost:3000/shop#)
[activesupport (7.0.3.1) lib/active_support/notifications.rb:206:in `instrument'](http://localhost:3000/shop#)
[actionview (7.0.3.1) lib/action_view/template.rb:361:in `instrument_render_template'](http://localhost:3000/shop#)
[actionview (7.0.3.1) lib/action_view/template.rb:155:in `render'](http://localhost:3000/shop#)
[actionview (7.0.3.1) lib/action_view/renderer/partial_renderer.rb:251:in `block in render_partial_template'](http://localhost:3000/shop#)
[activesupport (7.0.3.1) lib/active_support/notifications.rb:206:in `block in instrument'](http://localhost:3000/shop#)
[activesupport (7.0.3.1) lib/active_support/notifications/instrumenter.rb:24:in `instrument'](http://localhost:3000/shop#)
[activesupport (7.0.3.1) lib/active_support/notifications.rb:206:in `instrument'](http://localhost:3000/shop#)
[actionview (7.0.3.1) lib/action_view/renderer/partial_renderer.rb:246:in `render_partial_template'](http://localhost:3000/shop#)
[actionview (7.0.3.1) lib/action_view/renderer/partial_renderer.rb:237:in `render'](http://localhost:3000/shop#)
[actionview (7.0.3.1) lib/action_view/renderer/renderer.rb:81:in `render_partial_to_object'](http://localhost:3000/shop#)
[actionview (7.0.3.1) lib/action_view/renderer/renderer.rb:53:in `render_partial'](http://localhost:3000/shop#)
[actionview (7.0.3.1) lib/action_view/helpers/rendering_helper.rb:44:in `render'](http://localhost:3000/shop#)
[app/views/spree/home/index.html.erb:5](http://localhost:3000/shop#)
[actionview (7.0.3.1) lib/action_view/helpers/cache_helper.rb:175:in `cache'](http://localhost:3000/shop#)
[app/views/spree/home/index.html.erb:4](http://localhost:3000/shop#)
[actionview (7.0.3.1) lib/action_view/base.rb:244:in `public_send'](http://localhost:3000/shop#)
[actionview (7.0.3.1) lib/action_view/base.rb:244:in `_run'](http://localhost:3000/shop#)
[actionview (7.0.3.1) lib/action_view/template.rb:157:in `block in render'](http://localhost:3000/shop#)
[activesupport (7.0.3.1) lib/active_support/notifications.rb:206:in `block in instrument'](http://localhost:3000/shop#)
[activesupport (7.0.3.1) lib/active_support/notifications/instrumenter.rb:24:in `instrument'](http://localhost:3000/shop#)
[activesupport (7.0.3.1) lib/active_support/notifications.rb:206:in `instrument'](http://localhost:3000/shop#)
[actionview (7.0.3.1) lib/action_view/template.rb:361:in `instrument_render_template'](http://localhost:3000/shop#)
[actionview (7.0.3.1) lib/action_view/template.rb:155:in `render'](http://localhost:3000/shop#)
[actionview (7.0.3.1) lib/action_view/renderer/template_renderer.rb:65:in `block (2 levels) in render_template'](http://localhost:3000/shop#)
[activesupport (7.0.3.1) lib/active_support/notifications.rb:206:in `block in instrument'](http://localhost:3000/shop#)
[activesupport (7.0.3.1) lib/active_support/notifications/instrumenter.rb:24:in `instrument'](http://localhost:3000/shop#)
[activesupport (7.0.3.1) lib/active_support/notifications.rb:206:in `instrument'](http://localhost:3000/shop#)
[actionview (7.0.3.1) lib/action_view/renderer/template_renderer.rb:60:in `block in render_template'](http://localhost:3000/shop#)
[actionview (7.0.3.1) lib/action_view/renderer/template_renderer.rb:75:in `block in render_with_layout'](http://localhost:3000/shop#)
[activesupport (7.0.3.1) lib/active_support/notifications.rb:206:in `block in instrument'](http://localhost:3000/shop#)
[activesupport (7.0.3.1) lib/active_support/notifications/instrumenter.rb:24:in `instrument'](http://localhost:3000/shop#)
[activesupport (7.0.3.1) lib/active_support/notifications.rb:206:in `instrument'](http://localhost:3000/shop#)
[actionview (7.0.3.1) lib/action_view/renderer/template_renderer.rb:74:in `render_with_layout'](http://localhost:3000/shop#)
[actionview (7.0.3.1) lib/action_view/renderer/template_renderer.rb:59:in `render_template'](http://localhost:3000/shop#)
[actionview (7.0.3.1) lib/action_view/renderer/template_renderer.rb:11:in `render'](http://localhost:3000/shop#)
[actionview (7.0.3.1) lib/action_view/renderer/renderer.rb:61:in `render_template_to_object'](http://localhost:3000/shop#)
[actionview (7.0.3.1) lib/action_view/renderer/renderer.rb:29:in `render_to_object'](http://localhost:3000/shop#)
[actionview (7.0.3.1) lib/action_view/rendering.rb:117:in `block in _render_template'](http://localhost:3000/shop#)
[actionview (7.0.3.1) lib/action_view/base.rb:270:in `in_rendering_context'](http://localhost:3000/shop#)
[actionview (7.0.3.1) lib/action_view/rendering.rb:116:in `_render_template'](http://localhost:3000/shop#)
[actionpack (7.0.3.1) lib/action_controller/metal/streaming.rb:216:in `_render_template'](http://localhost:3000/shop#)
[actionview (7.0.3.1) lib/action_view/rendering.rb:103:in `render_to_body'](http://localhost:3000/shop#)
[actionpack (7.0.3.1) lib/action_controller/metal/rendering.rb:46:in `render_to_body'](http://localhost:3000/shop#)
[actionpack (7.0.3.1) lib/action_controller/metal/renderers.rb:141:in `render_to_body'](http://localhost:3000/shop#)
[actionpack (7.0.3.1) lib/abstract_controller/rendering.rb:25:in `render'](http://localhost:3000/shop#)
[actionpack (7.0.3.1) lib/action_controller/metal/rendering.rb:30:in `render'](http://localhost:3000/shop#)
[actionpack (7.0.3.1) lib/action_controller/metal/instrumentation.rb:22:in `block (2 levels) in render'](http://localhost:3000/shop#)
[/usr/local/rvm/rubies/ruby-3.1.2/lib/ruby/3.1.0/benchmark.rb:311:in `realtime'](http://localhost:3000/shop#)
[activesupport (7.0.3.1) lib/active_support/core_ext/benchmark.rb:14:in `ms'](http://localhost:3000/shop#)
[actionpack (7.0.3.1) lib/action_controller/metal/instrumentation.rb:22:in `block in render'](http://localhost:3000/shop#)
[actionpack (7.0.3.1) lib/action_controller/metal/instrumentation.rb:91:in `cleanup_view_runtime'](http://localhost:3000/shop#)
[activerecord (7.0.3.1) lib/active_record/railties/controller_runtime.rb:34:in `cleanup_view_runtime'](http://localhost:3000/shop#)
[actionpack (7.0.3.1) lib/action_controller/metal/instrumentation.rb:21:in `render'](http://localhost:3000/shop#)
[actionpack (7.0.3.1) lib/action_controller/metal/implicit_render.rb:35:in `default_render'](http://localhost:3000/shop#)
[actionpack (7.0.3.1) lib/action_controller/metal/basic_implicit_render.rb:6:in `block in send_action'](http://localhost:3000/shop#)
[<internal:kernel>:90:in `tap'](http://localhost:3000/shop#)
[actionpack (7.0.3.1) lib/action_controller/metal/basic_implicit_render.rb:6:in `send_action'](http://localhost:3000/shop#)
[actionpack (7.0.3.1) lib/abstract_controller/base.rb:215:in `process_action'](http://localhost:3000/shop#)
[actionpack (7.0.3.1) lib/action_controller/metal/rendering.rb:53:in `process_action'](http://localhost:3000/shop#)
[actionpack (7.0.3.1) lib/abstract_controller/callbacks.rb:234:in `block in process_action'](http://localhost:3000/shop#)
[activesupport (7.0.3.1) lib/active_support/callbacks.rb:118:in `block in run_callbacks'](http://localhost:3000/shop#)
[actiontext (7.0.3.1) lib/action_text/rendering.rb:20:in `with_renderer'](http://localhost:3000/shop#)
[actiontext (7.0.3.1) lib/action_text/engine.rb:69:in `block (4 levels) in <class:Engine>'](http://localhost:3000/shop#)
[activesupport (7.0.3.1) lib/active_support/callbacks.rb:127:in `instance_exec'](http://localhost:3000/shop#)
[activesupport (7.0.3.1) lib/active_support/callbacks.rb:127:in `block in run_callbacks'](http://localhost:3000/shop#)
[activesupport (7.0.3.1) lib/active_support/callbacks.rb:138:in `run_callbacks'](http://localhost:3000/shop#)
[actionpack (7.0.3.1) lib/abstract_controller/callbacks.rb:233:in `process_action'](http://localhost:3000/shop#)
[actionpack (7.0.3.1) lib/action_controller/metal/rescue.rb:22:in `process_action'](http://localhost:3000/shop#)
[actionpack (7.0.3.1) lib/action_controller/metal/instrumentation.rb:67:in `block in process_action'](http://localhost:3000/shop#)
[activesupport (7.0.3.1) lib/active_support/notifications.rb:206:in `block in instrument'](http://localhost:3000/shop#)
[activesupport (7.0.3.1) lib/active_support/notifications/instrumenter.rb:24:in `instrument'](http://localhost:3000/shop#)
[activesupport (7.0.3.1) lib/active_support/notifications.rb:206:in `instrument'](http://localhost:3000/shop#)
[actionpack (7.0.3.1) lib/action_controller/metal/instrumentation.rb:66:in `process_action'](http://localhost:3000/shop#)
[actionpack (7.0.3.1) lib/action_controller/metal/params_wrapper.rb:259:in `process_action'](http://localhost:3000/shop#)
[activerecord (7.0.3.1) lib/active_record/railties/controller_runtime.rb:27:in `process_action'](http://localhost:3000/shop#)
[actionpack (7.0.3.1) lib/abstract_controller/base.rb:151:in `process'](http://localhost:3000/shop#)
[actionview (7.0.3.1) lib/action_view/rendering.rb:39:in `process'](http://localhost:3000/shop#)
[actionpack (7.0.3.1) lib/action_controller/metal.rb:188:in `dispatch'](http://localhost:3000/shop#)
[actionpack (7.0.3.1) lib/action_controller/metal.rb:251:in `dispatch'](http://localhost:3000/shop#)
[actionpack (7.0.3.1) lib/action_dispatch/routing/route_set.rb:49:in `dispatch'](http://localhost:3000/shop#)
[actionpack (7.0.3.1) lib/action_dispatch/routing/route_set.rb:32:in `serve'](http://localhost:3000/shop#)
[actionpack (7.0.3.1) lib/action_dispatch/journey/router.rb:50:in `block in serve'](http://localhost:3000/shop#)
[actionpack (7.0.3.1) lib/action_dispatch/journey/router.rb:32:in `each'](http://localhost:3000/shop#)
[actionpack (7.0.3.1) lib/action_dispatch/journey/router.rb:32:in `serve'](http://localhost:3000/shop#)
[actionpack (7.0.3.1) lib/action_dispatch/routing/route_set.rb:852:in `call'](http://localhost:3000/shop#)
[railties (7.0.3.1) lib/rails/engine.rb:530:in `call'](http://localhost:3000/shop#)
[railties (7.0.3.1) lib/rails/railtie.rb:226:in `public_send'](http://localhost:3000/shop#)
[railties (7.0.3.1) lib/rails/railtie.rb:226:in `method_missing'](http://localhost:3000/shop#)
[actionpack (7.0.3.1) lib/action_dispatch/routing/mapper.rb:19:in `block in <class:Constraints>'](http://localhost:3000/shop#)
[actionpack (7.0.3.1) lib/action_dispatch/routing/mapper.rb:48:in `serve'](http://localhost:3000/shop#)
[actionpack (7.0.3.1) lib/action_dispatch/journey/router.rb:50:in `block in serve'](http://localhost:3000/shop#)
[actionpack (7.0.3.1) lib/action_dispatch/journey/router.rb:32:in `each'](http://localhost:3000/shop#)
[actionpack (7.0.3.1) lib/action_dispatch/journey/router.rb:32:in `serve'](http://localhost:3000/shop#)
[actionpack (7.0.3.1) lib/action_dispatch/routing/route_set.rb:852:in `call'](http://localhost:3000/shop#)
[actionpack (7.0.3.1) lib/action_dispatch/middleware/static.rb:23:in `call'](http://localhost:3000/shop#)
[warden (1.2.9) lib/warden/manager.rb:36:in `block in call'](http://localhost:3000/shop#)
[warden (1.2.9) lib/warden/manager.rb:34:in `catch'](http://localhost:3000/shop#)
[warden (1.2.9) lib/warden/manager.rb:34:in `call'](http://localhost:3000/shop#)
[rack (2.2.4) lib/rack/tempfile_reaper.rb:15:in `call'](http://localhost:3000/shop#)
[rack (2.2.4) lib/rack/etag.rb:27:in `call'](http://localhost:3000/shop#)
[rack (2.2.4) lib/rack/conditional_get.rb:27:in `call'](http://localhost:3000/shop#)
[rack (2.2.4) lib/rack/head.rb:12:in `call'](http://localhost:3000/shop#)
[actionpack (7.0.3.1) lib/action_dispatch/http/permissions_policy.rb:38:in `call'](http://localhost:3000/shop#)
[actionpack (7.0.3.1) lib/action_dispatch/http/content_security_policy.rb:36:in `call'](http://localhost:3000/shop#)
[rack (2.2.4) lib/rack/session/abstract/id.rb:266:in `context'](http://localhost:3000/shop#)
[rack (2.2.4) lib/rack/session/abstract/id.rb:260:in `call'](http://localhost:3000/shop#)
[actionpack (7.0.3.1) lib/action_dispatch/middleware/cookies.rb:697:in `call'](http://localhost:3000/shop#)
[activerecord (7.0.3.1) lib/active_record/migration.rb:603:in `call'](http://localhost:3000/shop#)
[actionpack (7.0.3.1) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'](http://localhost:3000/shop#)
[activesupport (7.0.3.1) lib/active_support/callbacks.rb:99:in `run_callbacks'](http://localhost:3000/shop#)
[actionpack (7.0.3.1) lib/action_dispatch/middleware/callbacks.rb:26:in `call'](http://localhost:3000/shop#)
[actionpack (7.0.3.1) lib/action_dispatch/middleware/executor.rb:14:in `call'](http://localhost:3000/shop#)
[actionpack (7.0.3.1) lib/action_dispatch/middleware/actionable_exceptions.rb:17:in `call'](http://localhost:3000/shop#)
[actionpack (7.0.3.1) lib/action_dispatch/middleware/debug_exceptions.rb:28:in `call'](http://localhost:3000/shop#)
[web-console (4.2.0) lib/web_console/middleware.rb:132:in `call_app'](http://localhost:3000/shop#)
[web-console (4.2.0) lib/web_console/middleware.rb:28:in `block in call'](http://localhost:3000/shop#)
[web-console (4.2.0) lib/web_console/middleware.rb:17:in `catch'](http://localhost:3000/shop#)
[web-console (4.2.0) lib/web_console/middleware.rb:17:in `call'](http://localhost:3000/shop#)
[actionpack (7.0.3.1) lib/action_dispatch/middleware/show_exceptions.rb:26:in `call'](http://localhost:3000/shop#)
[railties (7.0.3.1) lib/rails/rack/logger.rb:40:in `call_app'](http://localhost:3000/shop#)
[railties (7.0.3.1) lib/rails/rack/logger.rb:25:in `block in call'](http://localhost:3000/shop#)
[activesupport (7.0.3.1) lib/active_support/tagged_logging.rb:114:in `block in tagged'](http://localhost:3000/shop#)
[activesupport (7.0.3.1) lib/active_support/tagged_logging.rb:38:in `tagged'](http://localhost:3000/shop#)
[activesupport (7.0.3.1) lib/active_support/tagged_logging.rb:114:in `tagged'](http://localhost:3000/shop#)
[railties (7.0.3.1) lib/rails/rack/logger.rb:25:in `call'](http://localhost:3000/shop#)
[sprockets-rails (3.4.2) lib/sprockets/rails/quiet_assets.rb:13:in `call'](http://localhost:3000/shop#)
[actionpack (7.0.3.1) lib/action_dispatch/middleware/remote_ip.rb:93:in `call'](http://localhost:3000/shop#)
[actionpack (7.0.3.1) lib/action_dispatch/middleware/request_id.rb:26:in `call'](http://localhost:3000/shop#)
[rack (2.2.4) lib/rack/method_override.rb:24:in `call'](http://localhost:3000/shop#)
[rack (2.2.4) lib/rack/runtime.rb:22:in `call'](http://localhost:3000/shop#)
[activesupport (7.0.3.1) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'](http://localhost:3000/shop#)
[actionpack (7.0.3.1) lib/action_dispatch/middleware/server_timing.rb:20:in `call'](http://localhost:3000/shop#)
[actionpack (7.0.3.1) lib/action_dispatch/middleware/executor.rb:14:in `call'](http://localhost:3000/shop#)
[actionpack (7.0.3.1) lib/action_dispatch/middleware/static.rb:23:in `call'](http://localhost:3000/shop#)
[rack (2.2.4) lib/rack/sendfile.rb:110:in `call'](http://localhost:3000/shop#)
[actionpack (7.0.3.1) lib/action_dispatch/middleware/host_authorization.rb:137:in `call'](http://localhost:3000/shop#)
[railties (7.0.3.1) lib/rails/engine.rb:530:in `call'](http://localhost:3000/shop#)
[puma (5.6.5) lib/puma/configuration.rb:252:in `call'](http://localhost:3000/shop#)
[puma (5.6.5) lib/puma/request.rb:77:in `block in handle_request'](http://localhost:3000/shop#)
[puma (5.6.5) lib/puma/thread_pool.rb:340:in `with_force_shutdown'](http://localhost:3000/shop#)
[puma (5.6.5) lib/puma/request.rb:76:in `handle_request'](http://localhost:3000/shop#)
[puma (5.6.5) lib/puma/server.rb:443:in `process_client'](http://localhost:3000/shop#)
[puma (5.6.5) lib/puma/thread_pool.rb:147:in `block in spawn_thread'](http://localhost:3000/shop#)

**Reply by jarednorman** [reply]
This looks like some issue with images, but I'm not sure what's going on.

**Comment by can-tls** [comment]
I gave up and did everything from scratch, created a blog and installed solidus, it works on my second app but I would still like to know why stuff like this happens and how to solve that my own, it seems pretty hard to debug since some of the functionality is happening outside my app, in the solidus "core" right?

**Reply by jarednorman** [reply]
I'm not exactly sure what cause the issue you ran into, but I will say this: being able to debug code in gems is table stakes for developing Solidus stores.

It will be well worth your time to learn to how to use debuggers (my personal preference is `pry` and `pry-stack_explorer`, but there are lots of good ones) to navigate code outside your application. By running a Solidus store, you're essentially making Solidus your application and customizing from there, so it's pretty much impossible to do anything non-trivial without getting in there. 🙂
